### DISCUSSION TITLE
Several issues

### DISCUSSION THREAD

**Discussion by CuriousAmateur** [telekom-security/tpotce#1252] [Q&A]
I have several problems I'm not really sure how to fix, any help or guidance would be greatly appreciated.
Firstly, I run a distributed version of the tpot on two machines, one hive one sensor. They are both VM's (not sure about the underlying OS/hypervisor, it's company issues I'm guessing Vsphere). They both have 2 cpu's and 8gb ram. I have successfully deployed the sensor.
<img width="720" alt="image" src="https://user-images.githubusercontent.com/22777615/214520540-d0c257e6-f169-4a3a-a8dd-e05118b453fd.png">
Not sure what is causing the issue with the high cpu usage, I've checked the previous posts about it but to no avail.

Second, in the discover tab on kibana all the logs have the tpot_hostname = hive. Is this normal operating procedure or is it supposed to be the hostname of the sensor?

Thirdly, I'd like to stop several of the docker services but unsuccessfully. I've tried commenting out the services in the tpot.yml file and with docker stop several times. All I get with the docker stop command is the tpot stopping (or rather just restarting) and the service/honeypot is still up. Commenting out in the tpot.yml file did nothing, even after a reboot.

Lastly, the tpot service on the hive is always showing inactive even when almost all of the honeypots are up and running.
![image](https://user-images.githubusercontent.com/22777615/214531074-b2be5312-9475-4970-b92f-ec6a96d8e2d6.png)

Edit: just found these questions

What version of the OS are you currently using lsb_release -a and uname -a?
No LSB modules are available.
Distributor ID: Debian
Description:    Debian GNU/Linux 11 (bullseye)
Release:        11
Codename:       bullseye

What T-Pot version are you currently using? T-Pot 22.04.0
What edition (Standard, Nextgen, etc.) of T-Pot are you running? Distributed
What architecture are you running on (i.e. hardware, cloud, VM, etc.)? VM
Did you have any problems during the install? If yes, please attach /install.log /install.err. No
How long has your installation been running? ~month
Did you install upgrades, packages or use the update script? No
Did you modify any scripts or configs? If yes, please attach the changes. No
Please provide a screenshot of glances and htop. - attached above
![image](https://user-images.githubusercontent.com/22777615/214532680-66667f4a-7f28-4a08-81b5-213e69188d31.png)
How much free disk space is available (df -h)?
![image](https://user-images.githubusercontent.com/22777615/214532839-5a57e3e7-4c2e-492f-afcb-8d7dbfced922.png)
What is the current container status (dps.sh)? Attached above
What is the status of the T-Pot service (systemctl status tpot)?
![image](https://user-images.githubusercontent.com/22777615/214533051-fd406e8d-6d97-4895-ba2e-7942bc1683a2.png)
What ports are being occupied?
![image](https://user-images.githubusercontent.com/22777615/214533173-f9e5427f-3fdf-4cd2-a084-7d2bda306dbb.png)
If a single container shows as DOWN you can run docker logs <container-name> for the latest log entries N/A

**Comment by t3chn0m4g3** [comment]
Please follow the [Troubleshooting](https://github.com/telekom-security/tpotce#troubleshooting) section to pinpoint what is going on.

**Reply by CuriousAmateur** [reply]
I don't understand, the troubleshooting section just says I'm not guaranteed support or help, just free to post here. I can't pinpoint what is going on, there are too many errors, that's why I posted here. Do you need additional information, I'm not sure I follow.

**Reply by t3chn0m4g3** [reply]
You did not follow the Troubleshooting section which is why I cannot make sense of the information you posted above. Hence I cannot help.

**Reply by CuriousAmateur** [reply]
There is nothing provided in the troubleshooting section.
![image](https://user-images.githubusercontent.com/22777615/214831699-a09f79b3-9910-4218-9d55-f8f1ba57173b.png)
Tell me what you need I'll post it. Truly sorry about this, I've only been using this system for a month.
